### PR TITLE
Added JAX distributed training guide.

### PR DESCRIPTION
**Pull request by hertschuh**

### CODE REVIEW

File: guides/distributed_training_with_jax.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+for epoch in range(num_epochs):
+    data_iter = iter(train_data)
+    for data in data_iter:
+        x, y = data
+        sharded_x = jax.device_put(x.numpy(), data_sharding)
+        loss_value, train_state = train_step(train_state, sharded_x, y.numpy())
+    print("Epoch", epoch, "loss:", loss_value)
+
+"""
+That's it!
```
Comment (fchollet) [inline guides/distributed_training_with_jax.py:240]: Please add content about reattaching the state back to the model, since it's not obvious in this case

**Review by fchollet — COMMENTED** [review]
Nice work!

File: guides/distributed_training_with_jax.py (hunk 1 above)
Comment (hertschuh) [reply guides/distributed_training_with_jax.py:240]: Oh, right, I missed this part. Added.

I lost access to this branch, I guess I need to recreate this PR via a fork, right?

Comment (hertschuh) [reply guides/distributed_training_with_jax.py:240]: Replaced with https://github.com/keras-team/keras-core/pull/464

**Pull request closed without merging**